templates/confirmation-email-sent.html

```diff
@@ -14,9 +14,11 @@
   <h1 data-qa="confirmation-text">{{ _("A confirmation email has been sent to {email}").format(email = content.email)  }}</h1>
  {% endcall %}
 
-  <p><strong>{{ _("Didn't receive an email?")  }}</strong></p>
-  <p>{{ _("It can take a few minutes for the email to arrive. If it doesn't arrive, check your junk mail, or you can
-    <a href='{send_confirmation_email_url}' id='send-another-email'>send another confirmation email</a>.</p>").format(send_confirmation_email_url = content.send_confirmation_email_url)  }}
+  {% if content.show_send_another_email_guidance == False %}
```

Comment:
I think the logic is a bit off here, as this should be:
```suggestion
  {% if content.show_send_another_email_guidance %}
```